doc/DeveloperGuide.md

```diff
@@ -87,6 +87,42 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+#### Use case: Sort persons
+
+**MSS**
+
+1. User requests to sort persons
+2. AddressBook shows a list of persons sorted in the alphabetical order of names.
+Use case ends.
+
+**Extensions**
+
+2a. The list is empty
+
+> Use case ends
+
+#### Use case: Rename tags
+
+**MSS**
+
+1. User requests to rename a particular tag
+2. AddressBook shows a list of persons with the tag
+3. AddressBook prompts user to enter new name for existing tag
```

Comment:
Use cases should depict the back and forth interaction between the user and the system. For every step the user does, the system should do something in return (and vice-versa).